packs/data/abomination-vaults-bestiary.db/bone-gladiator.json

```diff
@@ -328,7 +328,7 @@
             "privateNotes": "",
             "publicNotes": "",
             "source": {
-                "value": "Pathfinder Abomination Vaults Hardcover Compilation"
+                "value": "Pathfinder Abomination Vaults Compilation Hardcover"
```

Comment:
@silvative Do you have a preference?

Reply:
It's a good question, since there's one instance of each.

Reply:
I think hardcover compilation sounds more sensible.

Reply:
Agreed. Fixed.